src/encoder/encodervorbis.cpp

```diff
@@ -149,7 +155,7 @@ void EncoderVorbis::encodeBuffer(const CSAMPLE *samples, const int size) {
  *
  * Currently this method is used before init() once to save artist, title and album
 */
-void EncoderVorbis::updateMetaData(char* artist, char* title, char* album) {
+void EncoderVorbis::updateMetaData(const char* artist, const char* title, const char* album) {
```

Comment:
This is a potential lifetime problem.  We store a pointer to a char array that is owned elsewhere. 

We have also the issue that this is currently a latin1 string. I think it is best to pass here a "const QString&" and decide inside the decoder how to convert it to a supported encoding format.